packages/opentelemetry-core/src/utils/environment.ts

```diff
@@ -37,7 +37,7 @@ const ENVIRONMENT_NUMBERS: Partial<keyof ENVIRONMENT>[] = [
  */
 export const DEFAULT_ENVIRONMENT: Required<ENVIRONMENT> = {
   OTEL_NO_PATCH_MODULES: '',
-  OTEL_LOG_LEVEL: LogLevel.ERROR,
+  OTEL_LOG_LEVEL: LogLevel.INFO,
```

Comment:
are you sure about this? - it might show too many data in console

Reply:
It's true but this is based on #856 , has `ERROR` been agreed upon since as what should be the default level?

Reply:
info is correct for now. I think we were going to change it to warn when we go GA

Reply:
I think we should open an issue to change the log level to warn and add `release:required-for-ga` label.